run_python_script.sh

```diff
@@ -34,11 +34,14 @@ cd $SLURM_SUBMIT_DIR
 
 srun /bin/hostname
 
-
 module load plgrid/tools/python/3.8
 module load plgrid/apps/cuda/11.2
 
+python3.6 -m venv venv
+source venv/bin/activate
+
+pip install --upgrade pip
```

Comment:
To nie odnosi się do `pip` z python'a 2?

Reply:
niee, to instaluje jako pip3 w tym venvie który został aktywowany

Reply:
To pewnie to pip3 na dole już nie jest potrzebne

Reply:
![image](https://user-images.githubusercontent.com/21079319/135772205-9f7c8d96-f443-433b-8120-9def088e72fd.png)